### PR TITLE
Feature/san 5100 v2

### PR DESCRIPTION
**Pull request by GarageInc**
1. Preview chart
image

2. Small popup with "...More"
image

3. Signal settings step route(fastest way)
image

### CODE REVIEW

File: src/components/SignalCard/SignalCard.module.scss [hunk 1]
```diff
@@ -52,16 +52,76 @@
...
+}
+
+.popup {
+  position: absolute;
+  left: 0;
+  top: 25px;
+  background: var(--white);
+
+  /* N200 - Porcelain */
+  border: 1px solid var(--porcelain);
```
Comment (haritonasty) [inline src/components/SignalCard/SignalCard.module.scss]: LOC 73 - you can delete it

File: src/components/SignalCard/SignalCard.module.scss [hunk 2]
```diff
@@ -52,16 +52,76 @@
...
+
 .bottom {
   margin-top: 19px;
   display: flex;
-  justify-content: space-between;
 
   &__right {
     display: flex;
   }
 }
```
Comment (haritonasty) [inline src/components/SignalCard/SignalCard.module.scss]: LOC 91 - 98 are still used or no?

File: src/components/SignalCard/SignalCard.module.scss (hunk 1 above)
Comment (GarageInc) [reply src/components/SignalCard/SignalCard.module.scss]: done

File: src/components/SignalCard/SignalCard.module.scss (hunk 2 above)
Comment (GarageInc) [reply src/components/SignalCard/SignalCard.module.scss]: Will clear

**Pull request merged**